app/models/user.rb

```diff
@@ -72,26 +72,26 @@ def opposite_role
   end
 
   # Courtesy of https://gist.github.com/ivanoats/7076128
-  def self.from_omniauth(auth, role)
-    where(auth.slice(:provider, :uid)).first_or_create do |user|
-      user.provider = auth.provider
-      user.uid = auth.uid
-      user.first_name = auth.info.nickname
-      user.email = auth.info.email
-      user.role = role
-    end
-  end
-
-  def self.new_with_session(params, session)
-    if session['devise.user_attributes']
-      new(session['devise.user_attributes'], without_protection: true) do |u|
-        u.attributes = params
-        u.valid?
-      end
-    else
-      super
-    end
-  end
+  # def self.from_omniauth(auth, role)
+  #   where(provider: auth.provider, uid: auth.uid).first_or_create do |user|
+  #     user.provider = auth.provider
+  #     user.uid = auth.uid
+  #     user.first_name = auth.info.nickname
+  #     user.email = auth.info.email
+  #     user.role = role
+  #   end
+  # end
+  #
+  # def self.new_with_session(params, session)
+  #   if session['devise.user_attributes']
+  #     new(session['devise.user_attributes'], without_protection: true) do |u|
+  #       u.attributes = params
+  #       u.valid?
+  #     end
+  #   else
+  #     super
+  #   end
+  # end
```

Comment:
Similarly... Ok to delete? It's in version control if we need to look back to it.